azure-pipelines.yml

```diff
@@ -0,0 +1,165 @@
+trigger:
```

Comment:
```
  - checkout: self
    fetchDepth: 2 # commit is referenced as HEAD^2 so minimal depth is 2
```

Dokud je to repo malé tak je to asi jedno, ale třeba u connection je to dramatický rozdíl :)

Reply:
Tak u connection je spíš primární problém ta miliarda zbytečných tagů, které teď nemažeme, ne? Proto ten checkout trvá a toto ho značně naboostuje.

Reply:
Jeden tag === label (tj. bajty). Ten depth 2 vyřeší to že se stahne jen to tolik binárních dat, kolik je potřeba na zhmotnění obsahu dvou commitů a nic víc... může tam být tagů miliarda a nic :)